pylintrc_backup

```diff
@@ -0,0 +1,508 @@
+# ***************************
```

Comment:
Did you create this file or did it come from edx_lint?

Reply:
You know it comes from edx-lint but I think at least half of the repos have this in their .gitignore or something because it never appeared in my testing. I can remove it.

Reply:
Thanks. This exercise has been a great way to uncover all the different tooling quirks that different repos have lol